cypress/support/routeSupport.js

```diff
@@ -50,7 +50,7 @@ Cypress.Commands.add("searchRouteWithAvoidFerryOption", isResponsive => {
 	cy.wait(2000);
 	cy.get('[placeholder="To"]').click().type("port said", { delay: 200 });
 	cy.contains("Port Said").click();
-	isResponsive ? cy.get('[data-testid="more-action-icon-container"]').click() : cy.contains("Route Options").click();
+	isResponsive ? cy.get('[data-testid="more-action-icon-container"]').click() : cy.contains("Avoid").click();
```

Comment:
Same as above: Can the cypress identifier you are looking for be more specific to elements specific to AvoidToll or is this more than that?